packages/jazz-tools/src/tools/tests/coList.test.ts

```diff
@@ -906,6 +907,29 @@ describe("CoList unique methods", () => {
     expect(result?.[2]).toBe("item3");
   });
 
+  test("upsertUnique without an active account", async () => {
+    const account = activeAccountContext.get();
+    activeAccountContext.set(null);
```

Comment:
I recently added a `runWithoutActiveAccount` helper that sets the active account to null and restores it afterward. It'd be nice to use it here